### ISSUE TITLE
[improvement suggestion] Error message when signing in

### ISSUE THREAD

**Issue by simonaszilinskas** [OpenSourcePolitics/decidim-module-friendly_signup#12]
Currently, when an incorrect email is entered, the error message is "Incorrect email or password". 
Instead of that I suggest to do a custom error message when the email doesn't exist saying "The email you've entered is not associated with an account. **Create a new [instance name] account.** " (the last sentence redirecting to the sign up page).

**Comment by Quentinchampenois** [comment]
I agree it can be confusing for user. However there is also a security scope to keep in mind : 
- For a pentester, if you try to guess email accounts, you can refer to the message to know if the email account exist. In this case, as a pentester, I will try to have a different message than `The email you've entered is not associated with an account`, then I "just" have to brute force password.
